src/main/java/com/scottlogic/SortStrategy.java

```diff
@@ -0,0 +1,15 @@
+package com.scottlogic;
+
+import java.util.List;
+
+public class SortStrategy {
```

Comment:
The strategy is usually the interface,

ie

Interface Sort
class BubbleSort implements SortStrategy
class InsertionSort implements SortStrategy